src/actions/write/extendRecord.ts

```diff
@@ -44,7 +75,7 @@ export const extendRecord = async (
   }
 
   // Check if it includes a valid number of years
-  if (!Number.isInteger(years) || years > MAX_YEARS) {
+  if (!Number.isInteger(years) || years > MAX_YEARS || years < MIN_YEARS) {
```

Comment:
we shouldn't need this check any more with AJV validations

Reply:
removed